app/js/tests/validation_dispatchers/test_address_validation_dispatcher.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+var test = require( 'tape' ),
+	sinon = require( 'sinon' ),
+	Actions = require( '../../lib/actions' ),
+	createAddressValidationDispatcher = require( '../../lib/validation_dispatchers/address' );
+
+test( 'AddressValidationDispatcher calls validator', function ( t ) {
+	var successResult = { status: 'OK' },
+		initialData = {},
+		testData = { street: 'The Yellow Brick Road', city: 'Emerald City', ignoredData: 'this won\'t be validated' },
+		expectedData = { street: 'The Yellow Brick Road', city: 'Emerald City' },
+		validator = { validate: sinon.stub().returns( successResult ) },
+		testStore = { dispatch: sinon.stub() },
+		dispatcher = createAddressValidationDispatcher(
+			validator,
+			initialData
+		);
+
+	dispatcher.dispatchIfChanged( testData, testStore );
+
+	t.ok( validator.validate.calledOnce, 'validation function is called once' );
+	t.ok( validator.validate.calledWith( expectedData ), 'validation function is called with selected fields' );
+	t.end();
+} );
+
+test( 'AddressValidationDispatcher dispatches result as action', function ( t ) {
+	var successResult = { status: 'OK' },
+		initialData = {},
+		testData = { street: 'The Yellow Brick Road', city: 'Emerald City', ignoredData: 'this won\'t be validated' },
+		validator = { validate: sinon.stub().returns( successResult ) },
+		testStore = { dispatch: sinon.spy() },
+		dispatcher = createAddressValidationDispatcher(
+			validator,
+			initialData
+		);
+
+	dispatcher.dispatchIfChanged( testData, testStore );
+
+	t.ok( testStore.dispatch.calledWith( Actions.newFinishAddressValidationAction( successResult ) ) );
```

Comment:
I'd agree with you in other cases. In this case, actions are basically value objects and action creation functions like `newFinishAddressValidationAction` are constructors of value objects and those aren't tested, right?